src/cdk/rollup_globals.bzl

```diff
@@ -0,0 +1,13 @@
+load("//:packages.bzl", "CDK_PACKAGES")
+
+# Base rollup globals for dependencies and the root entry-point.
+CDK_ROLLUP_GLOBALS = {
+  'tslib': 'tslib',
+  'moment': 'moment',
```

Comment:
Is moment needed for cdk? I think it should only be required for material-moment-adapter